frontend/src/pages/register-page/index.js

```diff
@@ -1,21 +1,25 @@
 import "./index.css";
 import React from "react";
+import NavBar from '../../components/navbar/index'
 
 const RegisterPage = () => {
   return (
-    <div className="container">
-      <div className="boxContainer">
-        <div>
-          <h2>Add a new user</h2>
-          <form className="ui form">
-            <div className="field">
-              <label>Name</label>
-              <input type="text"/>
-            </div>
-          </form>
-          <br/>
+    <div>
+      <NavBar/>
+      <div className="container">
+        <div className="boxContainer">
           <div>
-            <button className="button">Add User</button>
+            <h2>Add a new user</h2>
+            <form className="ui form">
+              <div className="field">
+                <label>Name</label>
+                <input type="text"/>
+              </div>
+            </form>
+            <br/>
+            <div>
+              <button className="button">Add User</button>
```

Comment:
maybe update the classname to be more descriptive of the component. dont forget to update the style name in the css file as well
```suggestion
              <button className="addButton">Add User</button>
```